Framework/DataHandling/test/ISISDataArchiveTest.h

```diff
@@ -9,36 +9,177 @@
 
 #include <cxxtest/TestSuite.h>
 #include <fstream>
+#include <sstream>
 
 #include "MantidAPI/ArchiveSearchFactory.h"
 #include "MantidDataHandling/ISISDataArchive.h"
 
 using namespace Mantid::DataHandling;
 using namespace Mantid::API;
 
+namespace {
+#ifdef _WIN32
```

Comment:
We should avoid duplicating these strings. It's the type of thing that is ripe for forgetting you need to update both of them. I think in the calls below that need to differentiate against windows/unix can just use `#ifdef _WIN32` themselves rather than checking this string.

Reply:
Good point, this was left over from before I re-jigged the classes